packages/core/src/config/address.ts

```diff
@@ -6,7 +6,7 @@ export type Factory<event extends AbiEvent = AbiEvent> = {
   /** ABI event that announces the creation of a new instance of this contract. */
   event: event;
   /** Name of the factory event parameter that contains the new child contract address. */
-  parameter: Exclude<event["inputs"][number]["name"], undefined>;
+  parameterPath: string;
```

Comment:
Can be updated to avoid a breaking change, where `parameter` is deprecated and later removed